sycl/test/lit.cfg.py

```diff
@@ -151,11 +155,11 @@
         os.remove(xrt_lock)
     acc_run_substitute="env --unset=XCL_EMULATION_MODE " + acc_run_substitute
     # hw_emu is very slow so it has a higher timeout.
+    acc_run_substitute += "unshare -pc --kill-child "
```

Comment:
Do not use abbreviated options, specially with such complex commands without comment!